### PR TITLE
Update app icons to new blue color

### LINKED ISSUE #14028: [Colors] Update app icons to use new Blue instead of WordPress Blue
To ensure that we are up to date with the [latest best practices of the Color Studio palette](https://github.com/Automattic/color-studio/blob/master/CHANGELOG.md), we should update the existing app icons to use new Blue instead of WordPress Blue. This will require new png assets for us to swap out in the app.

### PR DESCRIPTION
**Pull request by frosty**
Fixes #14028. This PR updates the app's icons to use the new blue color we're using in the rest of the UI.

**To test**

- Build and run, and verify the icons have the new light appearance on high resolution and low resolution, and look correct.

## Regression Notes
1. Potential unintended areas of impact

None, just updating resources.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/res/drawable/app_icon_foreground.xml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+      android:strokeWidth="1"
+      android:fillColor="#FFFFFF"
+      android:fillType="evenOdd"
+      android:strokeColor="#00000000"/>
```
Comment (malinajirka) [inline WordPress/src/main/res/drawable/app_icon_foreground.xml]:
```suggestion
      android:strokeColor="@android:color/transparent"/>
```

File: WordPress/src/main/res/drawable/app_icon_foreground.xml [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+      android:strokeWidth="1"
+      android:fillColor="#FFFFFF"
```
Comment (malinajirka) [inline WordPress/src/main/res/drawable/app_icon_foreground.xml]:
```suggestion
      android:fillColor="@color/white"
```

File: WordPress/src/main/res/drawable/app_icon_background.xml [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
-    </path>
-
+    android:height="108dp"
+    android:viewportWidth="432"
+    android:viewportHeight="432">
+  <path
+      android:pathData="M0,0h432v432h-432z"
+      android:strokeWidth="1"
+      android:fillType="evenOdd"
+      android:strokeColor="#00000000">
```
Comment (malinajirka) [inline WordPress/src/main/res/drawable/app_icon_background.xml]:
```suggestion
      android:strokeColor="@android:color/transparent">
```

File: WordPress/src/main/res/drawable/app_icon_background.xml [hunk 4]
```diff
@@ -1,13 +1,24 @@
...
+    <aapt:attr name="android:fillColor">
+      <gradient 
+          android:startY="0"
+          android:startX="133.87831"
+          android:endY="435.6443"
+          android:endX="293.90695"
+          android:type="linear">
+        <item android:offset="0" android:color="#FF399CE3"/>
```
Comment (malinajirka) [inline WordPress/src/main/res/drawable/app_icon_background.xml]:
```suggestion
        <item android:offset="0" android:color="@color/primary_30"/>
```

File: WordPress/src/main/res/drawable/app_icon_background.xml [hunk 5]
```diff
@@ -1,13 +1,24 @@
...
+        <item android:offset="1" android:color="#FF0675C4"/>
```
Comment (malinajirka) [inline WordPress/src/main/res/drawable/app_icon_background.xml]:
```suggestion
        <item android:offset="1" android:color="@color/primary_50"/>
```

**Review by malinajirka — CHANGES_REQUESTED** [review]
Thanks @frosty! The approach looks good to me overall. I added a few suggestions - they are all related to the same thing. We usually prefer to use color references instead of hardcoding colors into the drawable. For example, if we use "primary_30" and "primary_50" for the gradient, even if we change our theme colors in the future the icons will get updated automatically.

I'm still testing the changes, but I wanted to post this comments ASAP to speed things up.

**Comment by malinajirka** [comment]
Before:
| High resolution | Low resolution |
|---|---|
|  |  |

After:
| High resolution | Low resolution |
|---|---|
|  |  |

**Review by malinajirka — APPROVED** [review]
LGTM, feel free to merge this PR ;). Thanks

**Comment by frosty** [comment]
Thank _you_, @malinajirka!

**Comment by mattmiklic** [comment]
Looks great; thanks @frosty and thanks @malinajirka for the screenshots!

**Pull request merged**